src/docs/sdk/data-handling.mdx

```diff
@@ -1,23 +1,44 @@
 ---
 title: "Data Handling"
-sidebar_order: 5
+sidebar_order: 3
 ---
 
 Data handling is the standardized context in how we want SDKs help users filter data.
 
 ## Sensitive Data
 
-In older SDKs you might sometimes see elaborate constructs to allow the user to strip away sensitive data. Newer SDKs no longer have this feature as it turned out to be too hard to maintain per-SDK. Instead, only two simple config options are left:
+SDKs should not include PII or other sensitive data in the payload by default.
 
-- [_send-default-pii_](https://docs.sentry.io/error-reporting/configuration/#send-default-pii) is **disabled by default**, meaning that data that is naturally sensitive is not sent by default. That means, for example:
+- [_send-default-pii_](https://docs.sentry.io/platforms/python/configuration/options/#send-default-pii) is **disabled by default**, meaning that data that is naturally sensitive is not sent by default. That means, for example:
 
   - When attaching HTTP requests to events, "raw" bodies (bodies which cannot be parsed as JSON or formdata) are removed, and known sensitive headers such as `Authorization` or `Cookies` are removed too.
 
   - User-specific information (e.g. the current user ID according to the used webframework) is not sent at all.
 
-  - Note that if a user explicitly sets a request on the scope, nothing is stripped from that request. The above rules only apply to integrations that come with the SDK.
+  - On desktop applications, the username logged in the device is not included.
 
-- [_before-send_](https://docs.sentry.io/error-reporting/configuration/#before-send) can be used to register a callback with custom logic to remove sensitive data.
+  - SDKs don't send `{{auto}}` as `user.ip`. This instructs the server to keep the connection's IP address.
+
+*Note* that if a user explicitly sets a request on the scope, nothing is stripped from that request. The above rules only apply to integrations that come with the SDK.
+
+Before sending events to Sentry, the SDKs should invokes callbacks. That allows users to remove any sensitive data client-side.
+
+- `before-send` and `event-processors` can be used to register a callback with custom logic to remove sensitive data.
```

Comment:
links on these?

Reply:
We don't have generic docs (I'll use JS or Python as I've done in the other examples).
Actually, I'll link within the develop docs